main.py

```diff
@@ -1,10 +1,46 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+    for nickname, player_data in players_data.items():
+        # Create or get race
+        race_data = player_data["race"]
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"],
+            defaults={"description": race_data.get("description", "")}
+        )
+
+        # Create or get skills for the race
+        for skill_data in race_data.get("skills", []):
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                defaults={"bonus": skill_data["bonus"], "race": race}
+            )
+
+        # Create or get guild (if exists)
+        guild_data = player_data.get("guild")
+        guild = None
+        if guild_data:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_data["name"],
+                defaults={"description": guild_data.get("description")}
```

Comment:
The 'description' field in the Guild model is defined as nullable. When using 'get_or_create', ensure that the 'defaults' dictionary handles the possibility of 'description' being None to avoid unexpected behavior.